client/src/components/App.js

```diff
@@ -11,6 +11,7 @@ import DiagnosticContainer from './DiagnosticContainer';
 import { BrowserRouter, Route, Switch } from 'react-router-dom'
 import { connect } from 'react-redux';
 import setLoginLoading from '../redux/actions/actions'
+import Map from './Map'
```

Comment:
Need to import this in `App.test.js` as well for tests to compile

Reply:
added this to App.test.js. tests are still failing.

Reply:
<img width="411" alt="Screen Shot 2020-03-23 at 7 26 36 PM" src="https://user-images.githubusercontent.com/15767602/77372557-41586980-6d3c-11ea-92e5-4c803b54ca2b.png">

I haven't worked with this package before so I'm not sure what exactly needs to be imported - I'm just following along with what Travis CI is telling me, it looks like it's looking for a different module